opm/parser/eclipse/EclipseState/Schedule/MSW/SegmentSet.cpp

```diff
@@ -21,6 +21,11 @@
 #include <cmath>
 #include <map>
 
+#ifdef _WIN32
+#define _USE_MATH_DEFINES 
+#include <math.h>
```

Comment:
the

```c++
#define _USE_MATH_DEFINES 
```

looks suspicious. (since that macro starts with an underscore, it is assumed as an internal one by the C standard, so it is not allowed to be defined by user code. some compilers may warn about this...)